### ISSUE TITLE
Peer dependencies do not allow patch versions of react-dom

### ISSUE THREAD

**Issue by JakeSidSmith** [hot-loader/react-dom#45]
Am I correct in thinking that this package is automatically published when new versions of `react`/`react-dom` are released?

Currently `react` is on version `17.0.1`, but there is only a `17.0.0` version of `@hot-loader/react-dom`, and since the peer dependency on `react` is strict, you get a warning when installing this package (if using the latest `react`):

```shell
npm WARN @hot-loader/react-dom@17.0.0 requires a peer of react@17.0.0 but none is installed. You must install peer dependencies yourself.
```

Is there a specific reason the peer dependency is so strict? Could this be loosened?

**Comment by theKashey** [comment]
> Am I correct in thinking that this package is automatically published when new versions of react/react-dom are released?

Not quite. It's still manual releases and 17.0.1 was released just now 🚀

**Comment by JakeSidSmith** [comment]
Is it totally necessary to release a new version for every patch? Surely you'd save a lot of effort by loosening the peer?

**Comment by theKashey** [comment]
That is not under my control. But you don't have to use this library at all, as long as it just applies react-hot-loader "patch" to react. You can do the same via webpack loader.

Another good option is to stop using deprecated React-Hot-Loader and switch to FastRefresh. This time it will not require any extra _hack_ for living.

**Comment by JakeSidSmith** [comment]
Thanks, I had no idea there was an alternative. I'll have a look. 😊

**Comment by G-Rath** [comment]
@theKashey fyi react 17.0.2 was released, so we need a new bump :)

**Comment by deadcoder0904** [comment]
@theKashey yep, a new release is appreciated. I can't do `npm i` without adding `--legacy-peer-deps`.
